.travis.yml

```diff
@@ -11,7 +11,7 @@ before_install:
 
 install:
   - bundle install
-  - bundle exec pod repo update
+  - bundle exec pod repo update --silent
```

Comment:
I think the log is too long and wasteful so added `--silent` flag, but if you consider it's required, I'll revert https://github.com/braintree/braintree_ios/pull/303/commits/8ec8cab9feb9b441a7c4b336a92309cb8a8c55db commit.